docs/architecture/README.md

```diff
@@ -25,12 +25,14 @@ To suggest an ADR, please make use of the [ADR template](./adr-template.md) prov
 
 ## Table of Contents
 
-| ADR \#                                             | Description                                           | Status   |
-|----------------------------------------------------|-------------------------------------------------------|----------|
-| [001](./adr-001-handler-implementation.md)         | IBC handlers implementation                           | Accepted |
-| [002](./adr-002-error.md)                          | Error Management                                      | Accepted |
-| [003](./adr-003-ics20-implementation.md)           | ICS20 implementation                                  | Accepted |
-| [004](./adr-004-light-client-crates-extraction.md) | Light client crates extraction                        | Accepted |
-| [005](./adr-005-handlers-redesign.md)              | Handlers validation and execution separation          | Accepted |
-| [006](./adr-006-upgrade-client-implementation.md)  | Chain and client upgradability                        | Accepted |
-| [007](./adr-007-light-client-contexts.md)          | Light client contexts                                 | Accepted |
+| ADR \#                                                  | Description                                  | Status   |
+| ------------------------------------------------------- | -------------------------------------------- | -------- |
+| [001](./adr-001-handler-implementation.md)              | IBC handlers implementation                  | Accepted |
+| [002](./adr-002-error.md)                               | Error Management                             | Accepted |
+| [003](./adr-003-ics20-implementation.md)                | ICS20 implementation                         | Accepted |
+| [004](./adr-004-light-client-crates-extraction.md)      | Light client crates extraction               | Accepted |
+| [005](./adr-005-handlers-redesign.md)                   | Handlers validation and execution separation | Accepted |
+| [006](./adr-006-upgrade-client-implementation.md)       | Chain and client upgradability               | Accepted |
+| [007](./adr-007-light-client-contexts.md)               | Light client contexts                        | Accepted |
+| [008](./adr-008-restructure-ibc-crate.md)               | Restructure `ibc` crate                      | Proposed |
+| [010](./adr-010-enable-standalone-ics02-integration.md) | Enable standalone `ics-02` integration       | Proposed |
```

Comment:
```suggestion
| [010](./adr-010-enable-standalone-ics02-integration.md) | Enable standalone `ics-02` integration       | Accepted |
```

The status in the ADR should get updated to `Accepted` as well.

Reply:
Shouldn't ADR 008 be accepted too?

Also, should I change the status inside the ADR files?

Reply:
Ahh. Right. The files should be updated too.

Reply:
1453ee0